server/chat-plugins/helptickets.ts

```diff
@@ -508,7 +516,9 @@ function checkIp(ip: string) {
 for (const room of Rooms.rooms.values()) {
 	if (!room.settings.isHelp || !room.game) continue;
 	const game = room.getGame(HelpTicket)!;
```

Comment:
```suggestion
	const game = room.getGame(HelpTicket);
```

Reply:
Unfortunately, this has to be asserted. The above if is an or, not an and. TS thinks it can be null still.

Reply:
gross